app/models/company.rb

```diff
@@ -21,11 +21,19 @@ def get_info
         compensationandbenefits_rating: result["compensationAndBenefitsRating"].to_f,
         careeropportunities_rating: result["careerOpportunitiesRating"].to_f,
         worklifebalance_rating: result["workLifeBalanceRating"].to_f,
-        recommendtofriends_rating: result["recommendToFriendRating"].to_f,
+        recommendtofriends_rating: result["recommendToFriendRating"].to_f
+        )
+    end
+
+    if result && result["ceo"]
```

Comment:
If someone has some good ideas about how to clean this up, I would :heart: to hear it!